.run/Production.run.xml

```diff
@@ -0,0 +1,6 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Production" type="FlutterRunConfigurationType" factoryName="Flutter">
+    <option name="filePath" value="$PROJECT_DIR$/lib/main_production.dart" />
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
last one that missed the eof line

Reply:
you do not fix this? 🐤

Reply:
This file is generated by Android Studio, so I think we should keep it lol 🤔